### DISCUSSION TITLE
How to undo?

### DISCUSSION THREAD

**Discussion by ghost** [simeononsecurity/Windows-Optimize-Harden-Debloat#99] [Q&A] [answered]
Hey, is there no function to undo changes? Whatever it did to my firefox sucks xD i need my passwort manager in firefox. i dont want it to be "supervised" but still be secured with whatever security extensions it did add.

**Comment by simeononsecurity** [comment, marked as answer]
> Hey, is there no function to undo changes? Whatever it did to my firefox sucks xD i need my passwort manager in firefox. i dont want it to be "supervised" but still be secured with whatever security extensions it did add.

That is what most applications say when you apply settings to them in this automated means. It's usually what you would do in an actual enterprise. 

As far as your password manager goes, it isn't all that difficult to modify browser settings. 
Follow the instructions here https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat#editing-policies-in-local-group-policy-after-the-fact and then open up gpedit on your system. You should be able to find the settings for Mozilla Firefox pretty easily and you can change what you'd like. 

Instructions are at https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/wiki/How-to-Reverse-the-Changes-performed-in-Windows%E2%80%90Optimize%E2%80%90Harden%E2%80%90Debloat to reverse

**Reply by ghost** [reply]
it sadly did not create any restore point. also restore points always brought back a half alive windows system for me. i dont know anything about scripts and whatever policiys are and how they work. so i guess i just reinstall everything. just wanted a cleaned system from windows trash spying in the background and software i dont need. somehow after that everything did get more complicated than before. i kinda missunderstood what your script does. my fault. sry :)

**Reply by simeononsecurity** [reply]
> it sadly did not create any restore point. also restore points always brought back a half alive windows system for me. i dont know anything about scripts and whatever policiys are and how they work. so i guess i just reinstall everything. just wanted a cleaned system from windows trash spying in the background and software i dont need. somehow after that everything did get more complicated than before. i kinda missunderstood what your script does. my fault. sry :)

We've tested things pretty carefully. So if it didn't create a restore point, it has something to do with your computers configuration. Unfortunately that does mean the best course of action is a windows reinstall for you.
But from the sounds of things what you were looking for is https://github.com/simeononsecurity/Windows-Optimize-Debloat which is the version of this that doesn't include any hardening and is much lighter on the configurations.
